src/adhocracy_core/adhocracy_core/sheets/polarization.py

```diff
@@ -0,0 +1,47 @@
+"""Polarization sheet."""
+import colander
+
+from adhocracy_core.sheets import sheet_meta
+from adhocracy_core.interfaces import IPredicateSheet
+from adhocracy_core.sheets import add_sheet_to_registry
+from adhocracy_core.schema import AdhocracySchemaNode
+
+
+class IPolarizable(IPredicateSheet):
```

Comment:
No need to use a `PredicateSheet` here, as the `refers_to` field from the `IComment` sheet *is* the object, and the creator of the comment is the subject.